lib/pacemaker/pcmk_status.c

```diff
@@ -219,6 +220,43 @@ pcmk__output_cluster_status(pcmk__output_t *out, stonith_t *st, cib_t *cib,
     return pcmk_rc_ok;
 }
 
+int
+pcmk_status(xmlNodePtr *xml)
+{
+    cib_t *cib = NULL;
+    pcmk__output_t *out = NULL;
+    int rc = pcmk_rc_ok;
+
+    uint32_t show_opts = pcmk_show_pending | pcmk_show_inactive_rscs | pcmk_show_timing;
+
+#ifdef PCMK__COMPAT_2_0
+    show_opts |= pcmk_show_failed_detail;
+#endif
```

Comment:
We can drop this since this function didn't exist in 2.0. I don't think it matters for XML anyway

Reply:
Removed.